src/models/Account.ts

```diff
@@ -3,6 +3,11 @@ import mongoose from 'mongoose';
 import { encryptString } from '../util/encrypt';
 
 export interface IAccount {
+    firstName: string;
+    lastName: string;
+    type: string;
```

Comment:
@GarfDev I guess this is meant for the Individual/Organisation part, but that only has a purpose in the UI and should maybe be removed at all, since we can derive that information from the `organisation` property being set.

Lets remove `type` from storage:)